ansible_base/jwt_consumer/common/auth.py

```diff
@@ -250,26 +244,27 @@ def get_or_create_resource(self, content_type: str, data: dict) -> Tuple[Optiona
         if content_type == 'team':
             # For a team we first have to make sure the org is there
             org_id = data['org']
-            organization_data = self.token['objects'][self.org_content_type.model][org_id]
+            organization_data = self.token['objects']["organization"][org_id]
 
             # Now that we have the org we can build a team
-            resource, org = self.get_or_create_resource(self.org_content_type.model, organization_data)
-            team, created = self.team_content_type.model_class().objects.get_or_create(
-                name=data['name'], organization=org, defaults={'description': 'Waiting for resource sync'}
+            org_resource, _ = self.get_or_create_resource("organization", organization_data)
+
+            resource = Resource.create_resource(
```

Comment:
The comment before going into this block says:

> `# The resource was missing so we need to create its stub`

But `create_resource` does `content_object = processor(ObjModel()).save(resource_data, is_new=True)` which seems fairly specific that it is _creating a new content object_.

Should the comment say that both the resource & content_object need to be created? Or does this block of code need additional handling for updating the `ansible_id` of an existing item in some circumstances, and creating the content_object in other circumstances?